src/Chunking.cpp

```diff
@@ -75,11 +82,82 @@ void Chunking::untrackEntity(ChunkPos chunkPos, const EntityRef ref) {
         deleteChunk(chunkPos);
 }
 
+template<class T, size_t N>
+static void sendAroundPacket(const EntityRef recipient, std::vector<Bucket<T, N>>& buckets, uint32_t packetId) {
+    assert(recipient.kind == EntityKind::PLAYER);
+
+    uint8_t pktBuf[CN_PACKET_BUFFER_SIZE];
+    for (const auto& bucket : buckets) {
+        memset(pktBuf, 0, CN_PACKET_BUFFER_SIZE);
```

Comment:
This should be `CN_PACKET_BUFFER_SIZE - sizeof(int32_t)` here (and in the `pktBuf` declaration), because `CNSocket::sendPacket()` prepends the packet length to its buffer, which is only `CN_PACKET_BUFFER_SIZE` large. Same for the other function.

Reply:
uh oh, we use this same buffer size in the skill result handler, and probably other places.. I'll go around and fix em

Reply:
That could possibly be the root cause of rare bugs with mobs not being cleaned up. I'm not sure if those are still a thing or not.

Might be a good idea to define a constant for the max *usable* body size, ie. `CN_PACKET_BUFFER_SIZE - sizeof(int32_t)`.